mirrord-layer/src/lib.rs

```diff
@@ -133,6 +140,17 @@ fn init() {
     ));
 }
 
+fn should_load() -> bool {
+    let config = LayerConfig::init_from_env().unwrap();
+    if let Some(avoid_list) = config.skip_processes {
+        let args = std::env::args().collect::<Vec<_>>();
+        let process = args.first().unwrap().split('/').last().unwrap();
+        !avoid_list.split(';').any(|x| x == process)
```

Comment:
Stylewise, lets avoid non-meaningful variable names, even for iterators. This one could be named e.g. "process_to_avoid".

Reply:
@infiniteregrets I was referring to 'x', here